Dockerfile.dapper

```diff
@@ -29,8 +29,13 @@ RUN if [ "${ARCH}" == "amd64" ]; then \
         curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/spectrometer/master/install.sh | sh; \
```

Comment:
On testing, when there is a downstream cluster back in the integration tests, it would be good to check if that version is what is expected.